src/navigators/Navigator.tsx

```diff
@@ -129,7 +134,11 @@ class Navigator extends React.Component<Props, State> {
     }
 
     if (!__DEV__ && JailMonkey.isJailBroken()) {
-      return this.preventOpenAppWithRootedPhone();
+      isEmulator().then(isEmulator => {
+        if (!isEmulator) {
+          return this.preventOpenAppWithRootedPhone();
```

Comment:
it won't work as you intended this `return` is for `then` not for ` renderRoutes`

Reply:
you're right, my bad.